composer.json

```diff
@@ -75,7 +75,9 @@
             "Helhum\\Typo3Console\\Composer\\InstallerScripts::setVersion"
         ],
         "extension-copy-artifacts": [
-            "cp -f Resources/Private/ExtensionArtifacts/ext_* ."
+            "cp -f Resources/Private/ExtensionArtifacts/ext_* .",
+            "mkdir -p Resources/Public/Icons",
+            "cp -f Resources/Private/ExtensionArtifacts/ext_icon.svg Resources/Public/Icons/Extension.svg"
```

Comment:
Why not `cp -f ext_icon.svg Resources/Public/Icons/Extension.svg` here which was created in the first step?